pkg/cli/mirror/list/operators.go

```diff
@@ -145,25 +152,72 @@ func (o *OperatorsOptions) Run(cmd *cobra.Command) error {
 		if _, err := fmt.Fprintln(w, "Available OpenShift OperatorHub catalogs:"); err != nil {
 			return err
 		}
-		if err := o.writeIndexRef(w); err != nil {
+		if err := o.listCatalogs(w); err != nil {
 			return err
 		}
 	default:
-		return cmd.Help()
+
+		vm, err := getVersionMap(catalogs[0])
+		if err != nil {
+			return err
+		}
+
+		fmt.Fprintln(w, "Catalog versions:")
+
+		for v := range vm {
+
+			if _, err := fmt.Fprintf(w, "  %s\n", v); err != nil {
+				return err
+			}
+
+		}
 	}
 
 	return nil
 }
 
-func (o *OperatorsOptions) writeIndexRef(w io.Writer) error {
-	catalogs := []string{"redhat", "certified", "community"}
+var catalogs = []string{
+	"registry.redhat.io/redhat/redhat-operator-index",
+	"registry.redhat.io/redhat/certified-operator-index",
+	"registry.redhat.io/redhat/community-operator-index",
+	"registry.redhat.io/redhat/redhat-marketplace-index",
+}
+
+func (o *OperatorsOptions) listCatalogs(w io.Writer) error {
+
 	if _, err := fmt.Fprintf(w, "OpenShift %s:\n", o.Version); err != nil {
 		return err
 	}
 	for _, catalog := range catalogs {
-		if _, err := fmt.Fprintf(w, "registry.redhat.io/redhat/%s-operator-index:v%v\n", catalog, o.Version); err != nil {
-			return err
+		versions, err := getVersionMap(catalog)
+		if err != nil {
+			logrus.Error("Failed to get catalog version details: ", err)
+			continue
+		}
+
+		if versions["v"+o.Version] > 0 {
```

Comment:
I think listing the versions as a default output might be a bit unexpected. What are your thoughts on listing the four catalogs for the latest version of OpenShift as the default and then having another flag that lists the available catalog versions?